x-pack/test/functional/apps/lens/table.ts

```diff
@@ -56,17 +57,19 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     });
 
     it('should allow to configure column visibility', async () => {
+      screenshot.take('toggleColumnVisibility0: before');
```

Comment:
these and the x-pack/test/functional/page_objects/lens_page.ts screenshots are to be removed right? I think I left it there 🙈

Reply:
aww we were happy about them. Thank you again for all that you did for this maddening frustrating pr @mbondyra you rock!